src/modules/cmdpal/ExtensionTemplate/TemplateCmdPalExtension/Directory.Packages.props

```diff
@@ -10,6 +10,7 @@
     <PackageVersion Include="Microsoft.Windows.CsWinRT" Version="2.2.0" />
     <PackageVersion Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22621.2428" />
     <PackageVersion Include="Microsoft.WindowsAppSDK" Version="1.6.250205002" />
+    <PackageVersion Include="Shmuelie.WinRTServer" Version="2.1.1" />
```

Comment:
we need this mentioned in Notice.md

and potentially sign it as I assume sign check will fail. let's trigger the dart built to see what happens

Reply:
> let's trigger the dart built

After notice.md update, as that one will fail for sure

Reply:
oh ho ho, this project isn't actually in `PowerToys.sln`, so it doesn't get built by CI, so it doesn't trip NOTICE.md.

I ran a CI of this (and a bunch of others) over in https://dev.azure.com/microsoft/Dart/_build/results?buildId=118851971&view=results

Reply:
ha, I see. ok ok. :shipit:  then